code/script.js

```diff
@@ -1,9 +1,19 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
+const accordion = document.getElementsByClassName ('contentBx');
+
+for (i = 0; i<accordion.length; i++ ) {
```

Comment:
Maybe easier to read with som space between i<a ? Not sure, haven't read much JavaScript, not even sure that it works with space hehe

Reply:
I agree with the readability! I opened it up in VS code now to try it out and it works to put space in between! 👍🏽